pkg/nodejs/packagejson/testdata/parse_test.go

```diff
@@ -0,0 +1,73 @@
+package packagejson_test
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/nodejs/packagejson"
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+)
+
+func TestParse(t *testing.T) {
+	vectors := []struct {
+		file string // Test input file
+		want types.Library
+	}{
```

Comment:
I fixed this test before, but you did force push and my change was overwritten. Can you refer to this test?
https://github.com/aquasecurity/go-dep-parser/blob/dda1673181319dd5f216ba4324e75ab98bdc296f/pkg/ruby/gemspec/parse_test.go#L15-L20

Reply:
Updated similar to mentioned testcases.